htdocs/product/stock/class/mouvementstock.class.php

```diff
@@ -110,457 +110,458 @@ public function __construct($db)
 	 *	@param		string	$batch			batch number
 	 *	@param		boolean	$skip_batch		If set to true, stock movement is done without impacting batch record
 	 * 	@param		int		$id_product_batch	Id product_batch (when skip_batch is false and we already know which record of product_batch to use)
+	 *	@param		int		$excludesubproduct		Id of subproduct to exclude and avoid loop
```

Comment:
This parameter seems to not be used ?

Reply:
@eldy indeed, i forgot it :/ i have fix it